sql/augments.sql

```diff
@@ -348,9 +349,9 @@ INSERT INTO `augments` VALUES (276,0,0,0,0,0);
 INSERT INTO `augments` VALUES (277,0,0,0,0,0);
 -- End unused block
 
-INSERT INTO `augments` VALUES (278,0,412,1,0,0); -- Melee skill+1 (for automaton)
```

Comment:
gotta love when someone moves mod IDs and doesn't update it everywhere, leading to pets being granted "eat raw fish" :clap:

Reply:
Well, in a mithra's defense, they're just hungry....

Reply:
well when the automaton is eating all their fish, its no wonder!

Reply:
So looking at this section a second time, should the items for the automaton skills be assigned to the automaton or no?

Reply:
hmm I didn't think of that. Not sure what +skill items that aren't augments do exactly - whether thats on the master or the pet. Or if we have any +skill items for that matter. I guess we'll need to try it in game and see what happens.

Reply:
We do, I'll find them tomorrow and test them.

Reply:
So there aren't any items (found anywhere in the LSB repo) that have the ModIds for the automaton skills (ModIds 101 thru 103) assigned to them.  I then looked in petutils.cpp and found:

![image](https://user-images.githubusercontent.com/26195580/145229688-d49a5dcb-f6ff-4dab-ad1a-896340d38731.png)

whereas, for something like Blood Pact damage, you'll find (also in petutils.cpp):

![image](https://user-images.githubusercontent.com/26195580/145231747-04d2b7b8-2c59-4538-8d7c-2e953888def0.png)

So it does appear that having the automaton skills assigned to the Character and not the Pet is how it should be (right now, those automaton skills should _probably eventually_ be shifted to be assigned to the pet and not to the character).

Reply:
it likely has to be that way due to displaying in the master's status window on PUP

@claywar I summon thee

Reply:
@EpicTaru I had been thinking that rather than digging through code we just gm self into a pup give self an aug'd item, and _see what happens_

Reply:
I would assume we're assigning these mods to the character, since they're based on the gear that the character is wearing.  Also, there's nothing on the pet side to persist values, for example across a zone (as we're not storing current state in the DB).